linux_os/guide/system/software/updating/ensure_GPG_keys_are_configured/rule.yml

```diff
@@ -0,0 +1,34 @@
+documentation_complete: true
+
+prodtype: sle12,sle15
+
+title: 'Ensure GPG keys are configured'
+
+description: |-
+    Most packages managers implement GPG key signing to verify package integrity during
```

Comment:
I believe the rule description should be more elaborated. It should reflect the intention of the rule. Maybe you could provide some context. Also, it would be interesting to explicit this rule is not automated but limit to provide guidance to perform the manual check.

Reply:
@rumch-se , please, try the following steps:

1. ./build_product sle15
2. firefox build/guides/ssg-sle15-guide-cis.html
3. Search for "Ensure GPG keys are configured"
4. Take a look on how this rule description is presented to the users and how they would consume this information

If the purpose is to provide some guidance for the users, I believe it would be necessary to include at least the commands recommended to check this requirement manually, like presented in the `ocil`.

Also, the description is informing some generic information or some assumed preferred approaches, which is not wrong but seems too high-level and a little bit disconnected from `GPG keys` topic. I still miss the intention of the rule by reading the description. I would recommend to create a shorter and more objective description focusing on the GPG keys topic and provide some guidance.

In my personal assessment, thinking as an user/admin, if I read the `Ensure GPG keys are configured` title, I would expect to read the description to better understand:

- what is the GPG keys purpose, like in the last paragraph, which seems pretty good by the way
- why this can't be checked automatically
- how could I check it manually

This would provide all necessary information for a better user experience.